db/migrate/20210202052337_migrate_variant_unit_values.rb

```diff
@@ -1,14 +1,22 @@
 class MigrateVariantUnitValues < ActiveRecord::Migration
   def up
-    Spree::Variant.where(product_id: nil).destroy_all
-    Spree::Variant.where(unit_value: [nil, Float::NAN]).find_each do |variant|
-      variant.unit_value = 1
-      variant.save
```

Comment:
The Bugsnag errors indicate that there are actually some relevant callbacks in play here, and we potentially need them. See: `app/services/variant_units/option_value_namer.rb`. When this value changes, there are associated records that are also updated based on the new value.

Reply:
That was the original approach, here: https://github.com/openfoodfoundation/openfoodnetwork/pull/6841 ... I could be convinced that we should go that route still. It just seemed like it might be a long list of changes/added code for inconsistent variants that we maybe no longer care about.